packages/expo-video/android/src/main/java/expo/modules/video/VideoModule.kt

```diff
@@ -182,6 +184,35 @@ class VideoModule : Module() {
             ref.player.currentPosition
           }
         }
+        .set { ref: VideoPlayer, position: Long ->
+          appContext.mainQueue.launch {
+            ref.player.seekTo(position)
+          }
+          appContext.mainQueue.launch {
+            ref.player.currentPosition.toInt()
+          }
```

Comment:
```suggestion
```
Why do we have those lines? I think we can remove them, but correct me if I'm wrong.